src/Faker/Provider/Internet.php

```diff
@@ -45,23 +45,23 @@ public function email()
      */
     final public function safeEmail()
     {
-        return preg_replace('/\s/u', '', $this->userName() . '@' . static::safeEmailDomain());
+        return preg_replace('/\s/u', '', $this->userName().'@'.static::safeEmailDomain());
```

Comment:
Can you please add those spaces again for the concatenations? There is no defined PSR standard for concatenations, but keeping spaces around them improves readability. However, I agree to your other changes with short array syntax and additions of spaces.

Reply:
this is why style ci failed, i followed https://github.styleci.io/analyses/nN1MxO?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link which told me to do those changes. Who should I follow, you or style ci ?

Reply:
If style CI is suggesting that, I give way to the CI :)